wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -656,17 +653,36 @@ private void configPerModelSettings() {
         if (loadOnDevices == null) {
             loadOnDevices = prop.getProperty("load_on_devices", wlmc.getLoadOnDevices());
         }
+        if (engineName == null) {
+            engineName = inferEngine();
+        }
+
+        StringBuilder sb = new StringBuilder();
+        for (Map.Entry<Object, Object> entry : prop.entrySet()) {
+            String key = entry.getKey().toString();
+            if (!"job_queue_size".equals(key)
+                    && !"batch_size".equals(key)
+                    && !"max_idle_time".equals(key)
+                    && !"max_batch_delay".equals(key)
+                    && !"load_on_devices".equals(key)
+                    && !"engine".equals(key)
+                    && !"option.entryPoint".equals(key)) {
+                sb.append("\n\t").append(key).append(": ").append(entry.getValue());
+            }
+        }
+
         logger.info(
-                "Apply per model settings:\n\tqueueSize: {}\n\tbatchSize: {}"
-                        + "\n\tmaxBatchDelay: {}\n\tmaxIdle: {}\n\tloadOnDevices: {}"
-                        + "\n\tengine: {}\n\tentrypoint: {}",
+                "Apply per model settings:\n\tjob_queue_size: {}\n\tbatch_size: {}"
+                        + "\n\tmax_batch_delay: {}\n\tmax_idle_time: {}\n\tload_on_devices: {}"
+                        + "\n\tengine: {}\n\toption.entryPoint: {}{}",
```

Comment:
Why don't we add all of these with the string builder above? Is this just to preserve some order?

Reply:
They may not exists in serving.properties